.gitignore

```diff
@@ -62,7 +51,7 @@ jspm_packages
 .LSOverride
 
 # Icon must end with two \r
-Icon
+Icon

 
```

Comment:
Don't do that! Did you not see the comment? It has to end with those characters or it won't work.

Small value, since we don't support Windows.